components/exercises/chapter-3-oscar/Recursion.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import './Recursion.css';
```

Comment:
Do we need this CSS? can't we use Tailwind?

Reply:
Refactored